lib/rubocop/cop/mixin/regexp_metacharacter.rb

```diff
@@ -6,6 +6,10 @@ module Cop
     module RegexpMetacharacter
       private
 
+      def literal?(regex_str)
+        regex_str.match?(/\A#{Util::LITERAL_REGEX}+\z/)
+      end
```

Comment:
It may be moved to `Performance/StringInclude` if this method is not used for other than `Performance/StringInclude`.